internal/cmd/users.go

```diff
@@ -198,7 +183,7 @@ func CreateUser(req *api.CreateUserRequest) (*api.CreateUserResponse, error) {
 	return resp, nil
 }
 
-func UpdateUser(name string, cmdOptions editUserCmdOptions) error {
```

Comment:
Should this be called `updateUserPassword` ? Seems like it can only update that one field now.

Reply:
Will update this in a follow up PR. There already exists a `updateUserPassword` which #1999 is removing